Makefile

```diff
@@ -3,7 +3,10 @@
 build:
 	docker build --cache-from registry.heroku.com/robinho/web:latest -t registry.heroku.com/robinho/web .
 
-test: build
+stylecheck: build
+	docker run --rm registry.heroku.com/robinho/web flake8 robinho/
```

Comment:
could you add this to the circleci pipeline as well? https://github.com/fake-news-detector/robinho/blob/master/.circleci/config.yml#L16

we actually don't use `make` on the CI because it is not available for the docker CIs, the only available thing is docker itself

Reply:
Yep, I'll try to do this tonight :)

Reply:
And it's done.